packages/attribute-registry-process/src/services/attributeRegistryService.ts

```diff
@@ -217,115 +107,185 @@ export function attributeRegistryServiceBuilder(
       logger.info(`Retrieving attributes in bulk by id in [${ids}]`);
       return await readModelService.getAttributesByIds({ ids, offset, limit });
     },
-  };
-}
 
-export type AttributeRegistryService = ReturnType<
-  typeof attributeRegistryServiceBuilder
->;
+    async createDeclaredAttribute(
+      apiDeclaredAttributeSeed: ApiDeclaredAttributeSeed,
+      authData: AuthData
+    ): Promise<Attribute> {
+      logger.info(
+        `Creating declared attribute with name ${apiDeclaredAttributeSeed.name}}`
+      );
+      if (authData.externalId.origin !== "IPA") {
```

Comment:
I know you know, but when solving conflicts remember that this condition has been changed in the other PR